QuizApp/QuizSession/ViewModel/QuizSessionViewModel.swift

```diff
@@ -68,4 +73,15 @@ class QuizSessionViewModel {
             result: Result(numOfCorrectQuestions: numOfCorrectQuestions, numOfQuestions: questions.count))
     }
 
+    private func subscriptions() {
```

Comment:
```suggestion
    private func setBindings() {
```

Rename func name to smth actionable